assignment6/src/controller/shoppingCart.ts

```diff
@@ -0,0 +1,34 @@
+import { Request, Response } from "express";
+import { AppDataSource } from "../config/db";
+import { ShoppingCart } from '../entities/ShoppingCart';
+
+
+const addToShoppingCart = async (req: Request, res: Response) => {
+  const { userId, bookId } = req.params;
```

Comment:
there must be a middileware that will authorize user by userid and get user id from there rather than getting into params